### ISSUE TITLE
Could not evaluate some items: Resonators, Scarabs

### ISSUE THREAD

**Issue by ChrisFiftyTwo** [Kyusung4698/PoE-Overlay#5]
While trying to price check items I was unable to price check some items and it gives me this message: "Copied item could not be parsed. Make sure you have the correct language selected."

So far it has failed on:
Delve resonators
Scarabs

**Comment by josemariafs** [comment]
Also there are a issue with blighted maps (it search blight active gem)

**Comment by Kyusung4698** [comment]
Can you please share the copied Item on those? (CTRL+C while hovering) Thank you. I will look into it.

**Comment by josemariafs** [comment]
`Rarity: Normal
Gilded Perandus Scarab
--------
Area contains 4 additional Perandus Chests
Area contains Cadiro Perandus
--------
The dark day has come, Icius Perandus. Today either we succeed or humanity dies.
Enact your plan. Prevent the fate of the Vaal from befalling the Eternal Empire!
--------
Can be used in a personal Map Device to add modifiers to a Map.
`


`Rarity: Currency
Powerful Chaotic Resonator
--------
Stack Size: 5/10
Requires 3 Socketed Fossils
--------
Sockets: D D D 
--------
Reforges a rare item with new random modifiers
--------
All sockets must be filled with Fossils before this item can be used.
Shift click to unstack.
`

`Rarity: Normal
Gilded Perandus Scarab
--------
Area contains 4 additional Perandus Chests
Area contains Cadiro Perandus
--------
The dark day has come, Icius Perandus. Today either we succeed or humanity dies.
Enact your plan. Prevent the fate of the Vaal from befalling the Eternal Empire!
--------
Can be used in a personal Map Device to add modifiers to a Map.
`
